oidc-client.cabal

```diff
@@ -95,38 +94,34 @@ test-suite oidc-client-spec
     , http-client
     , http-client-tls
     , aeson
-    , scientific
     , jose-jwt
-    , crypton
     , exceptions
     , time
     , network-uri
 
 executable scotty-example
+  import:           warnings
+  if !flag(build-examples)
+    buildable: False
   main-is:          Main.hs
   default-language: Haskell2010
   hs-source-dirs:   examples/scotty/
-  ghc-options:      -Wall
-  if flag(build-examples)
-    build-depends:
-        base                >=4.7 && <5
-      , oidc-client
-      , aeson
-      , bytestring
-      , text
-      , containers
-      , transformers
-      , mtl
-      , wai-extra
-      , scotty
-      , scotty-cookie
-      , blaze-html
-      , cprng-aes
-      , crypto-random
-      , base64-bytestring
-      , http-types
-      , http-client
-      , tls                 >=1.3.2
-      , http-client-tls
-  else
-    buildable: False
+  build-depends:
+      base                >=4.7 && <5
+    , oidc-client
+    , aeson
+    , bytestring
+    , text
+    , containers
+    , mtl
+    , wai-extra
+    -- scotty-cookie not compatible with scotty 0.20+
+    , scotty < 0.20
```

Comment:
Had to add this upper bound - viz https://github.com/mmartin/scotty-cookie/issues/1

Reply:
The maintainer archived scrotty-cookie repo, because the functionality provided by that package has been merged into scotty proper in recent versions of scotty.